src/js/utils/hooks.js

```diff
@@ -42,3 +42,41 @@ export const useDidUpdateEffect = (onUpdate, deps) => {
         firstRef.current = true;
     }, deps);
 };
+
+export function getUrlSearchParams(key) {
+    const params = new URLSearchParams(window.location.search);
+    return params.get(key) || "";
+}
+
+export function updateUrlSearchParams(newTerm, key) {
```

Comment:
You write this generically for setting any search parameter, but it uses variable naming specific to the `term` use case.

```suggestion
export function updateUrlSearchParams(value, key) {
```